dark-theme.css

```diff
@@ -4344,6 +4344,10 @@ a.c-member_slug, .c-member_slug--link, .c-mrkdwn__subteam--link {
   color: var(--main-text);
 }
 
+.p-edit_profile__rules {
+  color: #949494;
```

Comment:
I notice this color is used a fair number of places, it would be nicer to push this to a variable.

Reply:
Yes I know. I haven't had time to add it to vars.